cPP_DSC.adoc

```diff
@@ -2326,7 +2338,9 @@ In addition to using the ST to demonstrate that ASE_TSS.1 has been satisfied, th
 
 === ADV: Development
 :xrefstyle: short
+
 The design information about the TOE is contained in the guidance documentation available to the end user as well as the TSS portion of the ST, and any additional information required by this cPP that is not to be made public (e.g., Entropy Essay). The DSC cPP requires only basic functional specification of interfaces presented in the AGD documentation (see <<Basic Functional Specification (ADV_FSP.1)>>) and specification of interfaces that can be invoked by a dependent component in a composed evaluation where the DSC is the base component.
+
 :xrefstyle: full
```

Comment:
This `:xrefstyle: full` was visible in the PDF. I hope the spacing will fix this, but I did not test it.

Reply:
#261